modules/developers-guide/pages/tutorials/scalability-cancan.adoc

```diff
@@ -0,0 +1,103 @@
+= Creating scalable apps
+ifdef::env-github,env-browser[:outfilesuffix:.adoc]
+:toc:
+:toc: right
+:toc-title: TUTORIAL - SCALABILITY
+:toclevels: 3
+:proglang: Motoko
+:platform: Internet Computer platform
+:IC: Internet Computer
+:company-id: DFINITY
+:sdk-short-name: DFINITY Canister SDK
+:sdk-long-name: DFINITY Canister Software Development Kit (SDK)
+
+The CanCan sample application is a simplified video-sharing service that demonstrates several features that you can use as models for your own applications.
+For example, here are a few things you can learn by exploring the CanCan sample application:
+
+* How to build a scalable application by splitting content into fragments for upload and storage then using queries to retrieve and reassemble the fragments for efficient streaming.
+
+* How to configure interoperability for an application that uses canisters written in different back-end languages.
+
+* How to implement a basic access control model for authenticating which videos are owned by which end-users.
+
+* How to build a front-end that implements more sophisticated user interface features for desktop or mobile apps.
+
+== Splitting uploaded content into multiple canisters
+
+Because canisters are compiled WebAssembly modules, they have certain known limitations. 
+For example, Web Assembly modules have a maximum of 4GB for memory and an upper limit on the number of object calls allowed.
+
+For a video-sharing sample applications like CanCan, these limitations mean that multiple canisters are required and that data must be broken into smaller chunks for storage and retrieval.
+
+The initial attempt to build a scalable video-sharing service for the {IC} used a distributed hash table as a back-end service with simple get and put functions that distributed data—chunks of the video to be uploaded or streamed—into a predefined set of canisters.
+In the early phases of the project, this approach was sufficient for a proof-of-concept and verifying that the video data could be properly transcoded for storage and retrieval.
+However, the scalability of the application was limited because the distributed hash table (DHT) relied on a specific number of canisters that it could populate with data for storage and retrieve data from for viewing.
```

Comment:
Nit: Introduce DHT acronym above, when its expansion is first used.

Reply:
Actually, the real reason why we introduce `BigMap` as a replacement for the `DHT` is to reduce the complexity. The {IC} protocol provides us certain guarantees that are generally not available on the regular Internet. For instance, a DHT node may disappear or lose data at any time, and on the {IC} the platform ensures that the canisters stay up and running (the part that you describe later on, as I see).

Reply:
Modified description